src/central-submissions.rst

```diff
@@ -72,13 +78,19 @@ To find the Form submissions page, first find the form in the Form listings page
 
    .. image:: /img/central-submissions/listing.png
 
-The table preview you see here will show you the first ten fields of your survey and their results, with the latest submissions shown closest to the top. Any downloadable files will appear with a green download link you can use to directly download that media attachment. The submission's instance ID will always be shown at the right side of this preview table.
+The table preview you see here will at first show you the first ten fields of your survey and their results, with the latest submissions shown closest to the top. Any downloadable files will appear with a green download link you can use to directly download that media attachment. The submission's instance ID will always be shown at the right side of this preview table.
+
+If your form has more than ten fields, you can show more columns by accessing the :guilabel:`Columns shown` dropdown and checking the columns you wish to see. While the Columns shown pane is open, you can use your browser's search feature (usually Ctrl+F or Cmd+F) to search for particular column name if you have many.
```

Comment:
We ended up adding a search field to the dropdown itself, so the user actually shouldn't need to use Ctrl/Cmd+F.

Reply:
I did go back and forth on this. I think it's a useful tip that I wouldn't probably have thought to try. I'll change it to "the search box or your browser's search feature".